include/nil/actor/zk/snark/systems/plonk/placeholder/verifier.hpp

```diff
@@ -118,59 +115,54 @@ namespace nil {
                         typename policy_type::evaluation_map columns_at_y;
                         for (std::size_t i = 0; i < witness_columns; i++) {
                             std::size_t i_global_index = i;
```

Comment:
Can you check please - is `i_global_index` and `i` it's always the same thing? So we don't need on of them.